### DISCUSSION TITLE
Pinning pull requests

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17310] [Pull Requests]
Hi all!

#### 1. Concept
I would like to pin pull requests, so I could show the community the most important pull requests that should be kept or not.  Currently on GitHub, we can 'Pinning an issue' in the repository, but there are no 'Pinning pull requests'.

#### 2. ui/ux
![image](https://user-images.githubusercontent.com/95204402/170498322-3384dfe4-cc80-486d-9b01-3e27fc0b5976.png)

#### 3. Reasons
1. We can pin the pull request so we can highlight the most important pull requests as a timeline
2. I would like to help the open source community and GitHub, I would like to know if this idea is a good idea or not
3. Did not find information about this feature on GitHub or other enterprise platforms, open or closed with this feature
4. Maybe this feature is something innovative and maybe it can contribute to that

#### 4. References
- https://docs.github.com/en/enterprise-server@3.1/issues/tracking-your-work-with-issues/pinning-an-issue-to-your-repository

**Comment by juanoa** [comment]
This could also be used to pin the release PR (develop -> main). In my team, we use to create this PR as soon as possible.

**Reply by p0358** [reply]
or to merge latest translations from the web translation platform, a CI action could automatically pin such PRs so it never gets lost

**Reply by BoberMod** [reply]
I agree, it would be very useful to keep PRs like release develop -> prod.

**Reply by siczmj** [reply]
That’s exactly what I was looking for. We also open upcoming release PRs, and it would be great to have them shown first.

**Comment by yeikel** [comment]
The first point is very important as it helps to highlight controversial changes such as breaking changes

**Comment by buhtz** [comment]
It seems obvious that Microsoft has no interest in its users wishes.
You might look for a code hoster using Gitea or Forgejo because there you can pin PRs very easy.

**Comment by grishasobol** [comment]
I am agree with that proposal. It is very uncomfortable to work in a big projects with many issues and PRs

**Comment by devhindo** [comment]
this would add a lot of value! we may have branches that get treated as the main branch and we create new branches from them, so it would add a lot of value to have pinned PRs that can be accessed easily.
